fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -40,21 +40,26 @@
     @Configurable("thorntail.microprofile.jwtauth.token.issuedBy")
     private Defaultable<String> tokenIssuer = string("http://localhost");
 
-    @AttributeDocumentation("The public key of the JWT token signer. Can be prefixed 'file:' or 'classpath:' for key assets, otherwise the key contents are expected")
+    @AttributeDocumentation("The public key of the JWT token signer. Can be prefixed 'file:' or 'classpath:' to refer to external assets, but this is deprecated; use 'thorntail.microprofile.jwt.token.signer-pub-key-location' instead")
     @Configurable("thorntail.microprofile.jwt.token.signer-pub-key")
     @Configurable("thorntail.microprofile.jwtauth.token.signerPubKey")
     private String publicKey;
 
+    @AttributeDocumentation("Location of the public key of the JWT token signer. The key can be in PEM or JWK formats, in the latter case it can be part of the JWK Set")
```

Comment:
I meant in what format can the location be :-) It seems `file:` and `classpath:` should still work, also `https://` should work for the JWKS, is there something else?

Reply:
@Ladicek Oh I see, will update. If no scheme is used then it will be indirecty defaulted to the `classpath:` in smalrye-jwt. But as I mentioned earlier, `file:` is not yet supported with this property, so I need to do few more tweaks

Reply:
OK, thanks!